eventmesh-connectors/eventmesh-connector-file/src/main/resources/source-config.yml

```diff
@@ -14,14 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+pubSubConfig:
+    meshAddress: 127.0.0.1:10000
+    subject: TopicTest
+    idc: FT
+    env: PRD
+    group: fileSource
+    appId: 5032
+    userName: fileSourceUser
+    passWord: filePassWord
 connectorConfig:
     connectorName: fileSource
     nameserver: 127.0.0.1:9877
     topic: TopicTest
     commitOffsetIntervalMs: 5000
-    fileName: TopicTest-%s
-    filePath: TopicTest/%s
+    fileName: userFileName
+    filePath: userFilePath
```

Comment:
The fileName (e.g., myFile) and filePath (e.g., /foo/bar) can be integrated into filePath (/foo/bar/myFile).

myFile is a single file, and when it's large, the disk I/O may encounter performance issues. For now, it can be accepted.

Reply:
@HarshSawarkar What do you think of this comment?

>The fileName (e.g., myFile) and filePath (e.g., /foo/bar) can be integrated into filePath (/foo/bar/myFile)

Reply:
Updated. @Pil0tXia  Can you please check now?